src/LookupTable/components/ImportCsvModal.tsx

```diff
@@ -50,18 +49,16 @@ const ImportCsvModal: React.FC<any> = ({
     let formData: any = new FormData()
     formData.append('file', file)
 
-    const pluralTableName = pluralize.isPlural(tableName) ? tableName : pluralize.plural(tableName)
-
-    if (!tableStructureID) formData.append('tableName', pluralTableName)
+    if (!tableStructureID) formData.append('tableName', tableName)
```

Comment:
Got rid of all that plural nonsense -- tables are now saved as singular names, just like all the other tables in the database. However -- you *can* still name it a plural name if you want, just remember that Postgraphile expects singular named tables when generating the GraphQL schema.